### DISCUSSION TITLE
[Question][HW2] How can I download json files from mongodb?

### DISCUSSION THREAD

**Discussion by hsKim25** [prosyslab-classroom/cs348-information-security#509] [Q&A]
I tried with mongodump or mongofiles with host ip and username, password.

But results of both are 'Auth fail'.
How can I get the json files from mongodb?

**Comment by MagicaQuartet** [comment]
I logged in to vm from the host (running virtualbox) via ssh, accessed mongodb server and copied records.

**Comment by pacokwon** [comment]
The instruction contains a subtle typo, maybe try `mongo -u cs448 -p cs448 192.168.20.1:27017/amp` from the attack server.

As far as I'm concerned the original instruction did not have a space between `cs448` and `-p` (like `cs448-p`)
